pkg/tekton/metapipeline/metapipeline.go

```diff
@@ -216,14 +215,15 @@ func stepCreateTektonCRDs(params CRDCreationParameters) syntax.Step {
 	args = append(args, "--service-account", params.ServiceAccount)
 	args = append(args, "--source", params.SourceDir)
 	args = append(args, "--branch", params.BranchIdentifier)
+	args = append(args, "--build-number", params.BuildNumber)
 	if params.Context != "" {
 		args = append(args, "--context", params.Context)
 	}
-	if len(params.Labels) > 0 {
-		args = append(args, "--label", strings.Join(params.Labels, ","))
+	for _, l := range params.Labels {
```

Comment:
:+1, eventually I am wondering whether this should not be "passed" via the environment directly. Not sure what the best approach is. Passing command-line arguments as we do here makes things more explicit, on the other hand using env variables seems more idiomatic and probably more flexible. 

I think it is important though that essential/required values are passed the same way regardless of what we want to use.